src/train.py

```diff
@@ -7,25 +7,23 @@
 import yaml
 from dlsia.core.train_scripts import Trainer
 from dvclive import Live
+from tiled.client import from_uri
 from torchvision import transforms
 
-from network import build_network
-from parameters import (
+from .network import build_network
+from .parameters import (
```

Comment:
I ran into import issues of not finding relevant packages. Is there any reason we need to go to the parent directory?
```suggestion
from network import build_network
from parameters import (
```

Reply:
Interesting. I didn't change this. FWIW, single `.` is same directory, not parent. You're running into import issues in pytest? I just committed a change to the pytest configuration in `pyproject.toml`. Can you try again with this change?

Reply:
> Interesting. I didn't change this. FWIW, single `.` is same directory, not parent. You're running into import issues in pytest? I just committed a change to the pytest configuration in `pyproject.toml`. Can you try again with this change?

I was wondering that too, because I did not see this change when we were discussing on Tuesday. Maybe it is introduced with formatting or pre-commits? Likely not but I can't think of any other reasons at this moment.

And yes I did ran into import issues, not with pytest, but when I tested the training script on my local end (i.e. `make train_msdnet_customdil`). Traceback below as a reference:

```
Traceback (most recent call last):
  File "/Users/tibbers/MLExchange/mlex_dlsia_segmentation_prototype/src/train.py", line 13, in <module>
    from .network import build_network
ImportError: attempted relative import with no known parent package
```